scipy/optimize/tests/test__basinhopping.py

```diff
@@ -347,14 +347,14 @@ def setup_method(self):
         self.x0 = np.array(1)
         self.f0 = 0
 
-        minres = OptimizeResult()
+        minres = OptimizeResult(success=True)
```

Comment:
Here, we need to adjust the tests of the `Storage` and `Metropolis` classes because we've changed a bit about how they work.